modules/apps/login/login-web/src/main/resources/META-INF/resources/login.jsp

```diff
@@ -180,11 +180,13 @@
 			<%@ include file="/navigation.jspf" %>
 		</div>
 
-		<aui:script sandbox="<%= true %>">
+		<aui:script position="inline" sandbox="<%= true %>">
 			var form = document.getElementById('<portlet:namespace /><%= formName %>');
 
 			if (form) {
 				form.addEventListener('submit', (event) => {
+					event.preventDefault();
```

Comment:
Again, tell the browser not to submit the form. This is necessary because the `submitForm` call below is `async` which means it can be executed long after it is invoked and we don't want the browser to submit the form on its own during that time period.